client/src/components/graph/setupLasso.js

```diff
@@ -67,25 +78,33 @@ const Lasso = () => {
       }
     };
 
+    const handleCancel = () => {
+      lassoPath.remove();
+      closePath = closePath?.remove();
```

Comment:
The assignment is just to catch the case that closePath is nullish and `closePath?.remove()` is an r-value which needs to be both captured and used according to our ESLint rules.  

Ideally it would just be:
```suggestion
      closePath?.remove();
```

Reply:
I can swap out the optional chaining for an if statement, or add the two ESLint ignores needed